src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6160,4 +6160,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="OutIsNotValidForReturn" xml:space="preserve">
     <value>'RefKind.Out' is not a valid ref kind for a return type.</value>
   </data>
+  <data name="ERR_CannotConvertAddressOfToDelegate" xml:space="preserve">
+    <value>Cannot convert a &amp; method group '{0}' to delegate type '{0}'.</value>
```

Comment:
>convert a &amp; method group [](start = 18, length = 28)

This message uses "convert a & method group" while the message below uses "convert & method group". We should be consistent. Consider dropping the "a" since it's not clear whether it should be "a" or "an".

Reply:
I'll correct this in my IDE pr I have out now.

---
In reply to: [432172457](https://github.com/dotnet/roslyn/pull/44613#discussion_r432172457) [](ancestors = 432172457)